benchmarks/src/main/scala/zio/chunks/ArrayBenchmarks.scala

```diff
@@ -0,0 +1,96 @@
+package zio.chunks
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import zio.Chunk
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+class ArrayBenchmarks {
+
+  @Param(Array("100", "1000", "10000"))
+  var array: Array[Int] = _
+
+  var chunk: Chunk[Int] = _
+
+  var size: Int = _
+
+  @Setup(Level.Trial)
+  def setup() = {
+    array = Array.fill(0)(size)
+    chunk = Chunk.fromArray(array)
+  }
+
+  @Benchmark
+  def arrayFold(): Int =
+    array.sum
+
+  @Benchmark
+  def arrayMap(): Array[Int] = array.map(_ * 2)
+
+  @Benchmark
+  def arrayMapOptimized(): Array[Int] = {
+    val len = array.length // we can drop this as it is equal to size which we're setting
+
+    val mapped = Array.ofDim[Int](len)
+    var i      = 0
+
+    while (i < len) {
+      mapped(i) = array(i) * 2 // this is f in chunk code
+      i += 1
+    }
+
+    mapped
+  }
+  @Benchmark
+  def arrayFind(): Option[Int] = array.find(_ > 2)
```

Comment:
Let's align names:

- `arrayFind` to `find`
- `arrayFindOptimized` to `findOptimized`
- etc.